resources/Versions.xml

```diff
@@ -0,0 +1,41 @@
+<data>
+	<version Name="Rebirth">
+		<entities File="EntitiesRebirth.xml"/>
+		<entities File="CollectiblesRebirth.xml"/>
+		<stages File="StagesRebirth.xml"/>
+		<roomtypes File="RoomTypesRebirth.xml"/>
+		<roomtypes File="SecretRoomTypesOld.xml" IgnoreWith="Repentance"/>
```

Comment:
Hmm, I guess this isn't quite desirable since it means the original entry would have to have a name to be replaced. Something along these lines though, where a subsequent entry can redefine an older one from a mixin